dinner_party2.rb

```diff
@@ -0,0 +1,62 @@
+# Chris Hintz
+
+puts "Welcome the the Hintz Deluxe Dinner Party Planning App!"
+
+people =[]
```

Comment:
Code Climate complained about a space after the `=`, just for general code style it's good to always have spaces on each side of any operator/comparison.
```rb
people = []
```